Tests/PointFreeTests/AuthTests.swift

```diff
@@ -10,7 +10,7 @@ import Optics
 import SnapshotTesting
 import XCTest
 
-class AuthTests: TestCase {
+class AuthIntegrationTests: LiveDatabaseTestCase {
```

Comment:
small potential gotcha for having two test classes in a single file is that snapshots may overwrite each other if the test function has the same name. only the #file and #function is baked into the snapshot file name.

Reply:
Yeah...though I think it's OK since collisions will get caught in test runs?